appveyor.yml

```diff
@@ -1,32 +1,32 @@
-# Only test one combination: "Visual Studio 12 + Win64 + Debug + DLL". We can
-# test more combinations but AppVeyor just takes too long to finish (each
-# combination takes ~15mins).
-platform:
-  - Win64
-
-configuration:
-  - Debug
-
-environment:
-  matrix:
-    - language: cpp
-      BUILD_DLL: ON
-
-    - language: csharp
-
-install:
-  - ps: Start-FileDownload https://googlemock.googlecode.com/files/gmock-1.7.0.zip
-  - 7z x gmock-1.7.0.zip
-  - rename gmock-1.7.0 gmock
-
-before_build:
-  - if %platform%==Win32 set generator=Visual Studio 12
-  - if %platform%==Win64 set generator=Visual Studio 12 Win64
-  - if %platform%==Win32 set vcplatform=Win32
-  - if %platform%==Win64 set vcplatform=x64
-
-build_script:
-  - CALL appveyor.bat
-
-skip_commits:
-  message: /.*\[skip appveyor\].*/
+# Only test one combination: "Visual Studio 12 + Win64 + Debug + DLL". We can
```

Comment:
On Tue, Oct 6, 2015 at 1:29 PM, Jisi Liu <notifications@github.com> wrote:

> In appveyor.yml
> <https://github.com/google/protobuf/pull/849#discussion_r41317456>:
>
> > -  - ps: Start-FileDownload https://googlemock.googlecode.com/files/gmock-1.7.0.zip
> > -  - 7z x gmock-1.7.0.zip
> > -  - rename gmock-1.7.0 gmock
> > -
> > -before_build:
> > -  - if %platform%==Win32 set generator=Visual Studio 12
> > -  - if %platform%==Win64 set generator=Visual Studio 12 Win64
> > -  - if %platform%==Win32 set vcplatform=Win32
> > -  - if %platform%==Win64 set vcplatform=x64
> > -
> > -build_script:
> > -  - CALL appveyor.bat
> > -
> > -skip_commits:
> > -  message: /.*\[skip appveyor\].*/
> > +# Only test one combination: "Visual Studio 12 + Win64 + Debug + DLL". We can
>
> Shall we change the git line ending settings to normalize them?
>
Probably not. Currently all code under csharp directory are using DOS
line-endings and for Visual Studio users that makes more sense.


> … <#1503ed81ec10be12_>
> On Tue, Oct 6, 2015 at 1:21 PM Feng Xiao <notifications@github.com>
> wrote: In appveyor.yml <
> https://github.com/google/protobuf/pull/849#discussion_r41316464>: > - -
> ps: Start-FileDownload
> https://googlemock.googlecode.com/files/gmock-1.7.0.zip > - - 7z x
> gmock-1.7.0.zip > - - rename gmock-1.7.0 gmock > - > -before_build: > - -
> if %platform%==Win32 set generator=Visual Studio 12 > - - if
> %platform%==Win64 set generator=Visual Studio 12 Win64 > - - if
> %platform%==Win32 set vcplatform=Win32 > - - if %platform%==Win64 set
> vcplatform=x64 > - > -build_script: > - - CALL appveyor.bat > - >
> -skip_commits: > - message: /.*\[skip appveyor\].*/ > +# Only test one
> combination: "Visual Studio 12 + Win64 + Debug + DLL". We can Could you
> revert this file? I believe the only change is about line-endings. We
> should probably exclude this file from the down-integrate script. — Reply
> to this email directly or view it on GitHub <
> https://github.com/google/protobuf/pull/849/files#r41316464>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/protobuf/pull/849/files#r41317456>.
>